lib/ansible/modules/cloud/azure/azure_rm_storageaccount_info.py

```diff
@@ -407,8 +406,10 @@ def exec_module(self, **kwargs):
         filtered = self.filter_tag(results)
 
         if is_old_facts:
-            self.results['ansible_facts']['azure_storageaccounts'] = self.serialize(filtered)
-            self.results['ansible_facts']['storageaccounts'] = self.format_to_dict(filtered)
+            self.results['ansible_facts'] = {
+                'azure_storageaccounts': self.serialize(filtered),
+                'storageaccounts': self.format_to_dict(filtered),
+            }
         self.results['storageaccounts'] = self.format_to_dict(filtered)
```

Comment:
Should only be returned on _info calls.

Reply:
This one did not return `storageaccounts` in `stable-2.8`: https://github.com/ansible/ansible/blob/stable-2.8/lib/ansible/modules/cloud/azure/azure_rm_storageaccount_facts.py#L335

The Azure team apparently did decide to change that for 2.9.